oslo/meetup.yaml

```diff
@@ -0,0 +1,69 @@
+cfpLink: ""
+meetupID: Cloud-Native-and-Kubernetes-Oslo
+city: Oslo
+country: norway
+name: Cloud Native and Kubernetes Oslo
+photo: https://secure.meetupstatic.com/photos/event/9/3/1/3/highres_461497651.jpeg
+latitude: 59.8937806
+longitude: 10.6450355
+organizers:
+- samialajrami
+- audunstrand
+meetups:
```

Comment:
the only thing that should be present here is stuff that can't be autogenerated, as the agenda, sponsors and recording fields. All this data below is autogenerated, and should not be present in this file anymore.